README.md

```diff
@@ -28,20 +28,15 @@ Documentation is provided within the library and can be accessed when loading Op
 Our documentation assumes that you have working knowledge of the Modelica language, are familiar with a Modelica-compliant modeling and simulation environment, that you have proeficient knowledge on power system steady state analysis (i.e., the so called "power flow"), and knowledge on power system dynamic modeling (i.e., the called "transient" and "small-signal" stability). As such, the documentation is limited, and aims to provide very consice information for people who fulfill the requirements above. If you do not fulfill these requirements, we recommend that you first get acquainted with both Modelica and Power Systems.
 
 ## Citing OpenIPSL in Publications:
-If you use OpenIPSL in your work or research, all we ask you in exchange is that you **cite the reference publications**, according to your use.
-Please consult our publication list, located within the Users' Guide package, in the Publications page, for browsing the reference publications.
-
-Preferably, please cite the following paper:
-  - Maxime Baudette, Marcelo Castro, Tin Rabuzin, Jan Lavenius, Tetiana Bogodorova, Luigi Vanfretti, OpenIPSL: Open-Instance Power System Library — Update 1.5 to “iTesla Power Systems Library (iPSL): A Modelica library for phasor time-domain simulations”, SoftwareX, Volume 7, January–June 2018, Pages 34-36, ISSN 2352-7110, https://doi.org/10.1016/j.softx.2018.01.002
-    - Link to Publisher's Website: [SoftwareX](https://www.sciencedirect.com/science/article/pii/S2352711018300050)
+If you use OpenIPSL in your work or research, all we ask you in exchange is that you **cite the reference publications**, according to your use. Please consult our publication list, located within the Users' Guide package, in the Publications page, for browsing the reference publications. Preferrably, please cite this repository by using our preferred reference, as seen on GitHub GUI.
```

Comment:
```suggestion
If you use OpenIPSL in your work or research, all we ask you in exchange is that you **cite the reference publications**, according to your use. Please consult our publication list, located within the User's Guide package, in the Publications page, for browsing the reference publications. Preferrably, please cite this repository by using our preferred reference, as seen on GitHub GUI.
```

Reply:
Looks good now with only this one typo.

Reply:
It was not a typo. I used Users' Guide because the guide supposedly belongs to all users and not to one specific user. Anyway, I think the singular will do the job since it is [an abstract representation of all users](https://english.stackexchange.com/questions/36657/user-s-guide-vs-users-guide#:~:text=The%20difference%20is%20in%20the,belonging%20to%20all%20the%20users.).